DOCS/man/options.rst

```diff
@@ -379,11 +380,11 @@ Program Behavior
     options that contain the word ``scale``. The special string ``*`` lists
     all top-level options.
 
-``-v``
-    Increment verbosity level, one level for each ``-v`` found on the command
+``--v``
```

Comment:
Even if this works, the usual convention is that single letter arguments only have one dash.

Reply:
Alright, can undo.

Reply:
fine to merge with this change

Reply:
undoing that is . . .